src/app/api/tools/route.ts

```
@@ -0,0 +1,93 @@
+import { NextRequest, NextResponse } from 'next/server';
+import prisma from '@/lib/prisma';
+
+export async function GET(req: NextRequest) {
+  const searchParams = req.nextUrl.searchParams;
+  const reqPage = parseInt(searchParams.get('page') || '1', 10);
+  const page = isNaN(reqPage) || reqPage < 1 ? 1 : reqPage;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)